docs/products/postgresql/howto/pgbouncer-stats.md

```diff
@@ -2,43 +2,56 @@
 title: Access PgBouncer statistics
 ---
 
-PgBouncer is used at Aiven for
-[Connection pooling](/docs/products/postgresql/concepts/pg-connection-pooling).
+PgBouncer is used at Aiven for [connection pooling](/docs/products/postgresql/concepts/pg-connection-pooling).
 
 ## Get PgBouncer URL
 
 PgBouncer URL can be checked under **Pools** in [Aiven
 Console](https://console.aiven.io/) > your Aiven for PostgreSQL®
 service's page > **Pools** view (available from the sidebar).
 Alternatively, it can be extracted via
-[Aiven Command Line interface](/docs/tools/cli), using `jq` ([https://stedolan.github.io/jq/](https://stedolan.github.io/jq/)) to parse the
+[Aiven Command Line interface](/docs/tools/cli), using `jq`
+([https://stedolan.github.io/jq/](https://stedolan.github.io/jq/)) to parse the
```

Comment:
Let's avoid clear urls as they tend to be meaningless, we can just put the link on jq, wdyt?